computer_vision/random_erasing_augmentation.py

```diff
@@ -0,0 +1,100 @@
+import glob
+import os
+import random
+from string import ascii_lowercase, digits
+
+import cv2
+import numpy as np
+
```

Comment:
remove `import numpy as np` as it is unused

Reply:
I did it